app/commands/community_video/create.rb

```diff
@@ -15,6 +15,8 @@ def call
           exercise:
         }
         video.save!
+
+        Exercise::UpdateHasApproaches.defer(exercise) if exercise.present?
```

Comment:
Move this to replace the `Rails.logger.info` placeholder in `app/models/community_video.rb`